<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autoRig</class>
 <widget class="QMainWindow" name="autoRig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>autoRig</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QLineEdit" name="nameTxtField">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>20</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Biped Name</string>
        </property>
       </widget>
       <widget class="QPushButton" name="leftBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>420</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mirror Left</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rightBtn">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>420</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mirror Right</string>
        </property>
       </widget>
       <widget class="QPushButton" name="guideBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>411</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Guide</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rigBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>500</y>
          <width>411</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RIG</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="guidePathTxtField">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="saveGuideBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>411</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save Guides</string>
        </property>
       </widget>
       <widget class="QPushButton" name="browseGuidesBtn">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>260</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadGuideBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>411</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load Guides</string>
        </property>
       </widget>
       <widget class="QPushButton" name="toggleguideBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>411</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>toggle Guides</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
